### DISCUSSION TITLE
Allow Projectwide Refresh Intervall in App

### DISCUSSION THREAD

**Discussion by CHSchuepfer** [directus/directus#14324] [Feature Requests]
Hey everyone,

just had an idea when i was setting the Refresh Setting on a few Tables in the App for me.
That it would be nice to have the option as an admin to set the Refresh intervall for all Users.
Either on a per Table Basis or, as the Titles eludes to, Projectwide.

Users could change it of course but it would be nice to have a baseline set by the admin.

Thanks for the great work and so long

**Comment by benhaynes** [comment]
@rijkvanzanten , is the refresh interval saved within the collection presets?

**Reply by rijkvanzanten** [reply]
I believe it is, yes

**Reply by benhaynes** [reply]
Then this should be possible now by setting a global preset! 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
